README.md

```diff
@@ -68,25 +68,27 @@ Optimization simply use Ax.
 
 The latest release of BoTorch is easily installed either via
 [Anaconda](https://www.anaconda.com/distribution/#download-section) (recommended):
+
+First install PyTorch:
+```bash
+# Needed for OSX only, see below.
+conda install pytorch torchvision -c pytorch 
+```
+
+_Recommendation for MacOS users:_ PyTorch is a required dependency of BoTorch, and can be automatically installed via `pip`. However, we recommend installing PyTorch manually (as above) so that PyTorch is properly linked against MKL (a library that optimizes mathematical computation for Intel processors). This will result in up to an order-of-magnitude speed-up for Bayesian optimization, as at the moment, installing PyTorch from pip does not link against MKL.
+
+If you need CUDA on MacOS, or want to customize your installation (i.e. CPU-only option), please follow the [PyTorch installation instructions](https://pytorch.org/get-started/locally/) to build from source.
+
+Then install BoTorch:
 ```bash
-conda install botorch -c pytorch -c gpytorch -c conda-forge
+conda install botorch -c conda-forge
```

Comment:
At the time the `conda-forge` channel was a fix to make sure `pyro-ppl` can be resolved (see https://github.com/pytorch/botorch/pull/1207). In general we have recommended people install gpytorch from the `gpytorch` and `botorch` from the `pytorch` channels, respectively, rather than relying on `conda-forge` (which may not have the latest packages).

However, if this copying of the package into `conda-forge` has been working reliably and most people use `conda-forge` anyway then we could consider just pointing to `conda-forge` to simplify the installation? 

```suggestion
conda install botorch -c pytorch -c gpytorch -c conda-forge
```

Reply:
yeah that seems right -- I thought that installing BoTorch from the pytorch channel rather than conda-forge fixed the MKL warning I was getting, but on second check I'm not sure that's right. I'll look into that more carefully and make sure I land on some install instructions that are actually working.